mnefun/_mnefun.py

```diff
@@ -1122,6 +1110,33 @@ def run_sss_locally(p, subjects, run_indices):
             raw_sss.save(o, overwrite=True, buffer_size_sec=None)
 
 
+def _load_trans_to(p, subj, run_indices, raw=None):
+    if isinstance(p.trans_to, string_types):
+        if p.trans_to == 'median':
+            trans_to = op.join(p.work_dir, subj, p.raw_dir,
+                               subj + '_median_pos.fif')
+            if not op.isfile(trans_to):
+                calc_median_hp(p, subj, trans_to, run_indices)
+        trans_to = mne.read_trans(trans_to)
+    elif p.trans_to is None:
+        trans_to = None if raw is None else raw.info['dev_head_t']
+    else:
+        trans_to = np.array(p.trans_to, float)
+        if trans_to.shape == (4,):
+            t = np.eye(4)
+            t[:3, 3] = trans_to[:3]
+            theta = np.deg2rad(trans_to[3])
+            t[1:3, 1:3] = [[np.cos(theta), -np.sin(theta)],
+                           [np.sin(theta), np.cos(theta)]]
+            trans_to = mne.Transform('meg', 'head', t)
+        elif trans_to.shape != (3,):
+            raise ValueError('trans_to must have 3 or 4 elements, '
+                             'got shape %s' % (trans_to.shape,))
+    if trans_to is not None:
+        trans_to = mne.transforms._ensure_trans(trans_to, 'meg', 'head')
```

Comment:
For some reason `_ensure_trans` is returning a 
>ValueError: trans must be a Transform between meg<->head, got None

when `params.trasn_to = (0, 0, .03)`